docs/site/_includes/getting_started/global/partials/INSTALL_OTHER.liquid

```diff
@@ -87,4 +87,15 @@ dhctl bootstrap-phase abort \
 
 After the installation is complete, you will be returned to the command line.
 
```

Comment:
We did not not find resources to install.
I think, we need to add ClusterAuthorizationRule and User resources here.
Without they, user can not see our dashboards.

Reply:
In this guide we use the Minimal bundle and enable only deckhouse-web (don't enable the user-authn module and dashboard). So, after install, the deckhouse-web UI can be accessible through the basic-auth. On the next step of the guide there is a command to get admin password to access to the deckhouse-web UI

Reply:
Ok. I got it